ElectronClient/gui/ResourceScreen.tsx

```diff
@@ -45,29 +52,35 @@ interface ActiveSorting {
 const ResourceTable: React.FC<ResourceTable> = (props: ResourceTable) => {
 	const sortOrderEngagedMarker = (s: SortingOrder) => {
 		return (
-			<a href="#" onClick={ () => props.onToggleSorting(s) }>{
-				(props.sorting.order === s && props.sorting.type === 'desc') ? '▾' : '▴' }</a>
+			<a href="#"
+				style={{ color: props.theme.htmlLinkColor }}
+				onClick={() => props.onToggleSorting(s)}>{
+					(props.sorting.order === s && props.sorting.type === 'desc') ? '▾' : '▴'}</a>
 		);
 	};
-	return <table style={{ width: '90%' }}>
+	return <table style={{ width: props.style.width - 40 }}>
 		<thead>
 			<tr>
-				<th>{_('Title')} { sortOrderEngagedMarker('name') }</th>
```

Comment:
The typical style in Joplin is to leave whitespace in. This file seems to be a mixed bag in that regard, so maybe laurent will need to comment on it.

Reply:
Not sure about the whitespace rule in this case, but as long as it passes the linter it's fine.